api/src/main/java/org/pmiops/workbench/BlockscoreService.java

```diff
@@ -1,11 +1,18 @@
 package org.pmiops.workbench;
 
+import com.blockscore.models.Address;
 import com.blockscore.models.PaginatedResult;
 import com.blockscore.models.Person;
+import org.pmiops.workbench.model.BlockscoreVerificationStatus;
 
 /**
  * Encapsulate API for interfacing with Blockscore identity verification service.
  */
 public interface BlockscoreService {
   public PaginatedResult<Person> listPeople();
+  public Person createPerson(
```

Comment:
Another option is returning some object that wraps Person and has the extracted BlockscoreVerificationStatus on it. (This is fine too.)

Reply:
So there is an `isValid` method which is what I should have been using. I basically converted validity to a boolean and then I don't have to use the private field anymore.